ci/destroy-aro-cluster.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
insert the same thoughts around bash strict mode here

Reply:
ACK